### ISSUE TITLE
the usage of batchnorm1d in self.att_embed of AttModel class

### ISSUE THREAD

**Issue by wzn0828** [ruotianluo/self-critical.pytorch#65]
In AttModel class, the usage of batchnorm1d in self.att_embed is not as the described in the official docs. I think there should be a view operation before batchnorm1d.

**Comment by ruotianluo** [comment]
https://github.com/ruotianluo/self-critical.pytorch/blob/master/models/AttModel.py#L104
check this.

**Comment by wzn0828** [comment]
Thank you very much.
